src/styles/index.scss

```diff
@@ -242,13 +242,6 @@ div[class*='sidebarViewport'] {
 
   h4 {
     margin-bottom: 32px;
-    color: var(--core-color-solid-slate-1400);
```

Comment:
any particular reason to remove style from global file ? is there any default `SCSS` applied from quill ?

Reply:
yes, quill got the default color

Reply:
what about font-size @agrim-deriv ? are we using default or have configure somewhere else ?

Reply:
we have default font-sizes as well in quill.

Reply:
In this case we are not using quill heading and text. As its our old code and we haven't updated there so I we remove this styling would that work fine as a before or it has a chance that these tags would have different behavior than quill `H4` and `Text` ?  
![image](https://github.com/user-attachments/assets/f7db711c-a2f7-4697-acdd-93a386646b7c)

Reply:
Nope the behavior is the same. Refer to https://quill-ui.pages.dev/?path=/docs/components-typography-heading--playground and to our figma https://www.figma.com/design/vxKXqGpgiwUpHnghfL4FjJ/API---App-Registration?node-id=10317-32846&node-type=instance&m=dev